### ISSUE TITLE
Clean url ssr not working after refresh on build

### ISSUE THREAD

**Issue by chenraz** [zeit/next.js#5353]
# Bug report

## Describe the bug

Clean url ssr are not rendered on server side, after running 'npm run build".

I had forked the learnnextjs repository - lean url ssr branch.
followed the "fetching data for pages" tutorial
https://nextjs.org/learn/basics/fetching-data-for-pages/setup

So far - so good, I can see the post on:
http://localhost:3000/p/975

Then, I followed the "Deploying a Next.js App" tutorial (without git checkout)

https://nextjs.org/learn/basics/deploying-a-nextjs-app

but, after "npm run build", and "npm run start", the page breaks:

http://localhost:3000/p/975 - 404

Instead, the page is visible on:

http://localhost:3000/post?id=975

## To Reproduce

I forked my app from the learnnextjs-demo to make it easier.

`git clone https://github.com/chenraz/learnnextjs-demo.git`
`cd learnnextjs-demo`
`npm install`
` npm run dev`

The app is working, I can directly access a post page on localhost:
http://localhost:3000/p/975

Then:

`npm run build`
`npm run start`

now on
 http://localhost:3000/p/975 
I am getting 404 (only if i click the link, navigation via front-end is no problem)

But, the page is visible on 
http://localhost:3000/post?id=975

You can see it on NOW deployment:

https://hello-next-iviwdciixk.now.sh/p/975 - 404 page
https://hello-next-iviwdciixk.now.sh/post?id=975 - working

## Expected behavior

A link for a post should work:
https://hello-next-iviwdciixk.now.sh/p/975

## System information
 - Windows 10 Pro - Version 1803
 - Version of Next.js: 7.0.1

**Comment by sergiodxa** [comment]
That's expected, `next start` will start the Next.js server not the custom server you build to have clean URLs.

Change `next start` to `NODE_ENV=production node server.js` as shown on https://nextjs.org/learn/basics/server-side-support-for-clean-urls/create-a-custom-server.

**Comment by chenraz** [comment]
Thanks for this impressively fast reply.

Actually I tried this tutorial before, but adding the script:
"start": "NODE_ENV=production node server.js"
through an error, so i tried other things.

Reading you comment I tried it again.
Turns out that because i am running on windows, I had to add to scritps:
"start": "node server.js"
and not
"start": "NODE_ENV=production node server.js"

and the, set it manually on powerShell:
`set NODE_ENV=production`
and only then:
` npm run start`

So now it is working perfectly well on deployment.

But now i am facing a different problem when trying to deploy it to NOW. Instead of getting the App, i am getting a folders and files:

https://learnnextjs-demo-ansahqdevm.now.sh/
